prow/gerrit/adapter/adapter.go

```diff
@@ -528,12 +548,8 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 		return "", nil
 	}
 
-	logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for preparing to process gerrit changes.")
```

Comment:
I see that we've removed the more detailed timing and logging of the separate parts of the `processChange` function. We may need some of these to determine why individual change processing time is high.

I think it is fine to remove them for now since we're cleaning up the log based metrics, but we may need to replace those with prometheus metrics in a follow up to get a better breakdown of the timing of `processChange`.

Reply:
Agreed; let's revisit this again if necessary in a future PR that adds more metrics.